README.md

```diff
@@ -21,8 +21,19 @@ To view a community members’ contributions, provide their full name, Gerrit us
 There is also a feature to upload a list of usernames in a CSV format and view contributions for several community members altogether (this feature is still in experimental mode). 
 
 ## Getting started
-* First, clone the repo `git clone https://github.com/wikimedia/WikiContrib.git`
-* This tool has two different components: Frontend & Backend. You will have to set up both for the tool to be fully functional, and the instructions do so are here:
+- First, clone the repo `git clone https://github.com/wikimedia/WikiContrib.git`
+
+#### Using Docker Compose
```

Comment:
These instructions are for running the project, right? And, for when one has already followed frontend and backend Docker instructions. This wasn't clear to me immediately, and maybe it should be made clear here, and links to separate installation guides should be added.

Reply:
Sure on it!